conn.c

```diff
@@ -33,13 +33,13 @@ make_conn(int fd, char start_state, tube use, tube watch)
     job j;
     Conn *c;
 
-    c = new(Conn);
-    if (!c) return twarn("OOM"), NULL;
+    c = (Conn *)new(Conn);
```

Comment:
I think the thing to do is put something in dat.h like this, with a comment

```
// work around unhelpful built-in NULL on some compilers
#ifndef NULL
#define NULL ((void*)0)
#endif
```

If that still gets the job done cleanly on your system, let's roll with it.